### DISCUSSION TITLE
How to access writable from app-service code

### DISCUSSION THREAD

**Discussion by herrdragon** [edgexfoundry/community#68] [Q&A] [answered]
Hello team,

Is there a way to access the Writable section in configuration.toml like ApplicationSettings? or is there another way to know If I change a setting in Consul, how can I get the new value in the code?

Thank you

**Comment by lenny-goodell** [comment, marked as answer]
@herrdragon , custom App Service can have structured custom config with own Writable section. Use this when the simpler ApplicationSettings isn't sufficient.

https://docs.edgexfoundry.org/2.3/microservices/application/AdvancedTopics/#structure-custom-configuration

**Reply by herrdragon** [reply]
Perfect, thank you very much. I will add the custom section
